compiler/utils/bitsets.nim

```diff
@@ -15,58 +15,108 @@ type
   TBitSet* = seq[ElemType]    # we use byte here to avoid issues with
                               # cross-compiling; uint would be more efficient
                               # however
+  TBitSetView* = openArray[ElemType] ## A view into a bit set. This doesn't
+                                     ## necessarily have to be a `TBitSet`,
+                                     ## but can be any byte sequence storing
+                                     ## representing a bit set
```

Comment:
If you want to alias `openArray[X]`, you have to write it out explicitly for now as using it like this, the compiler might interpret it as a concrete view type, which is an unstable feature.

Reply:
Even when only used as a parameter type? That's a bit annoying, but I'll use `openArray[ElemType]` for now then